container/docker/engine.py

```diff
@@ -768,6 +775,9 @@ def bootstrap_env(self, temp_dir, behavior, operation, compose_option,
                              builder_img_id=builder_img_id,
                              config=config_yaml,
                              env=os.environ,
+                             playbook=self.playbook,
+                             with_volumes=with_volumes,
```

Comment:
Since `with_volumes` and `with_variables` are in `self.params`, which is also passed as an argument, no need to repeat them here.